packages/docs/getting-started/hosted-nodes.md

```diff
@@ -0,0 +1,27 @@
+# Hosted nodes
+
+This page explains how to get a preconfigured Celo blockchain node running on one of the major cloud providers. cLabs currently provides machine images for launching full and lightest nodes on Alfajores and Mainnet, these prebuilt images are updated with every release of the Celo blockchain client and available on Amazon Web Services and Google Cloud Platform.
+
+Before proceeding with a hosted Celo blockchain node, you'll need to have an account with your cloud provider of choice and basic knowledge of networking.
+
+{% hint style="info" %}
+If you would like to keep up-to-date with all the news happening in the Celo community, including validation, node operation and governance, please sign up to our [Celo Signal mailing list here](https://celo.activehosted.com/f/15).
```

Comment:
nit: "sign up ~to~ for our [Celo Signal mailing list ~here~"

"here" is unnecessary.